src/Logging/HTTPOutputHandler.php

```diff
@@ -137,6 +138,13 @@ public function setDefaultFormatter(FormatterInterface $formatter)
         return $this;
     }
 
+    protected function shouldSuppressError(int $errorCode): bool
+    {
+        return $errorCode === E_USER_DEPRECATED
+            && Deprecation::triggeringError()
+            && (Director::is_cli() ? Deprecation::shouldSuppressForCli() : Deprecation::shouldSuppressForHttp());
```

Comment:
Lets through all errors that are _not_ `E_USER_DEPRECATED`, or which _are_ that but weren't emitted from the `Deprecation` class, or which _were_ emitted from there but shouldn't be suppressed per config.